core/src/test/java/nu/yona/server/sms/PlivoSmsServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package nu.yona.server.sms;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import nu.yona.server.properties.SmsProperties;
+import nu.yona.server.properties.YonaProperties;
+
+@RunWith(Parameterized.class)
```

Comment:
This parametrizes the complete class, so I think the class should be renamed to a more specific name. Or you could consider to use [https://github.com/Pragmatists/JUnitParams]